CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ librdkafka v2.0.2 is a bugfix release:
 * Fix OpenSSL version in Win32 nuget package (#4152).
 
 
+## Enhancements
```

Comment:
@edenhill I'm not sure if this matches what appears to be the new Changelog style post-2.0 - I can modify it if necessary.